app/controllers/v0/ask_va/static_data_auth_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'date'
+
+module V0
+  module AskVA
+    class StaticDataAuthController < ApplicationController
+      def index
+        data = {
+          Ruchi: { 'data-info': 'ruchi.shah@thoughtworks.com' },
+          Eddie: { 'data-info': 'eddie.otero@oddball.io' },
+          Jacob: { 'data-info': 'jacob@docme360.com' },
+          Joe: { 'data-info': 'joe.hall@thoughtworks.com' },
+          Khoa: { 'data-info': 'khoa.nguyen@oddball.io' }
+        }
+        render json: data, status: :ok
+      rescue => e
```

Comment:
Rescuing all exceptions becomes a platform issue, because you could swallow something that's not related to your code and we'd have a hard time finding out. Is there a specific exception or error you're guarding against here? If so, specify that please. If not, we need to raise the exception -- which you could still do at the end of `#service_exception_handler` -- but it's a platform risk to swallow everything.

Reply:
This is a mockup of our actual app. Just to have the app provisioned in the vets-website and vets-api, so not all cases were considered. But, you are 1,000% correct. We'll correct this before requesting another review.